packages/experimental/Badge/src/CounterBadge/CounterBadge.tsx

```diff
@@ -22,18 +22,24 @@ export const CounterBadge = compose<CounterBadgeType>({
     const iconProps = createIconProps(userProps.icon);
     const badge = useCounterBadge(userProps);
 
-    const Slots = useSlots(badge.props, (layer) => badgeLookup(layer, badge.props));
+    const Slots = useSlots(badge.props, (layer) => badgeLookup(layer, badge.props) || (layer === 'dot' && badge.props['dot']));
```

Comment:
badgeLookup already looks are userProps[layer], does that not work for props exclusive to counter badge?

Reply:
yes, you're right. Thanks for catching